test/ffmpeg-vaapi/decode/jpeg.py

```diff
@@ -23,6 +33,7 @@ def before(self):
   def test(self, case):
     vars(self).update(spec[case].copy())
     self.case = case
+    self.mformat = self.mapformat(self.format)
```

Comment:
This should be done in base class with default return value.  Then in this subclass, just override self.mapformat.